recipe_scrapers/mykoreankitchen.py

```diff
@@ -0,0 +1,89 @@
+# mypy: allow-untyped-defs
+
+from recipe_scrapers._grouping_utils import IngredientGroup
+
+from ._abstract import AbstractScraper
+
+
+class MyKoreanKitchen(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "mykoreankitchen.com"
+
+    def author(self):
+        return self.schema.author()
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
+        return self.schema.category()
+
+    def total_time(self):
+        return self.schema.total_time()
+
+    def yields(self):
+        return self.schema.yields()
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        return self.schema.ingredients()
+
+    def ingredient_groups(self):
+        ingredient_list = self.soup.find_all(
+            "div", {"class": "wprm-recipe-ingredient-group"}
+        )
+
+        ingredient_groups = []
+        ingredient_index_base = 0
```

Comment:
I'd probably suggest renaming `ingredient_groups` to `results` here, allowing `ingredient_list` to be renamed to the more-precise `ingredient_groups`.

Reply:
Sounds good to me.